data/mods/Magiclysm/monsters/holiday_magiclysm.json

```diff
@@ -0,0 +1,103 @@
+[
+  {
+    "id": "mon_yulecat",
+    "type": "MONSTER",
+    "name": "Yule Cat",
+    "description": " Magical beast from Icelandic folklore, known for their catlike appearance and their hobby of eating Workshop Elves.",
```

Comment:
Is the starting space here intentional or a typo?

Reply:
Typo.